src/interfaces/root-state-interface.ts

```diff
@@ -49,6 +49,8 @@ export interface RootStateIF {
   parties: Array<PartyIF>
   recordsAddress: OfficeAddressIF
   registeredAddress: OfficeAddressIF
-  tasks: Array<ApiTaskIF> // "tasks" data from API
+  // *** TODO: declare a type for pendingsList
+  pendingsList: Array<any> // pendings list from bootstrap filing
```

Comment:
I'll declare a type for this in my next PR as I keep working on the ticket.

Reply:
Sounds good